.coin-or/config.yml

```diff
@@ -22,8 +22,8 @@ Description:
 Dependencies:
   - Description: ThirdParty wrapper for building ASL
     URL: https://github.com/coin-or-tools/ThirdParty-ASL
-    Version: stable/2.0
-    Required: Optional
+    Version: master
+    Required: Required
```

Comment:
Why change to Required for ASL?

Reply:
It seemed the easiest way to get ASL used in the CI jobs. I don't intend to keep this change. That's why this PR is a draft.